src/document/DocumentAnnotator.ts

```diff
@@ -112,10 +111,10 @@ export default class DocumentAnnotator extends BaseAnnotator {
         }
 
         const canSmoothScroll = 'scrollBehavior' in this.annotatedEl.style;
-        const parentCenterX = this.annotatedEl.clientWidth / 2;
-        const parentCenterY = this.annotatedEl.clientHeight / 2;
-        const offsetCenterX = Math.round(offsetX * this.scale);
-        const offsetCenterY = Math.round(offsetY * this.scale);
+        const parentCenterX = Math.round(this.annotatedEl.clientWidth / 2);
+        const parentCenterY = Math.round(this.annotatedEl.clientHeight / 2);
+        const offsetCenterX = Math.round(pageEl.clientWidth * (offsetX / 100));
+        const offsetCenterY = Math.round(pageEl.clientHeight * (offsetY / 100)); // 500px * 0.01 = 5px offset
```

Comment:
What is the comment related to?

Reply:
I think I had it in here as a reminder to myself on how this works, since it's a little odd. I can remove it if it's not helpful.